packages/services/api/src/modules/schema/providers/schema-manager.ts

```diff
@@ -189,8 +184,6 @@ export class SchemaManager {
       };
     }
 
-    const orchestrator = this.matchOrchestrator(project.type);
```

Comment:
Should we offload the `'Only Federation projects are supported'` error logic to the `compositionOrchestrator`?
It will error if the number of schemas is > 1 and the project type is Single. So it already has some logic around what each projectType supports.
Right now all this feels a little confusing where we're mostly handling project type inside the orchestrator but still have these edge cases.